spring/src/test/java/org/axonframework/spring/config/AutoWiredStateStoredAggregateTest.java

```diff
@@ -41,13 +37,14 @@
 import javax.persistence.Id;
 
 import static org.axonframework.modelling.command.AggregateLifecycle.apply;
-import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.Mockito.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.Mockito.mock;
 
 @ExtendWith(SpringExtension.class)
 @EnableMBeanExport(registration = RegistrationPolicy.IGNORE_EXISTING)
 @ContextConfiguration
-public class AutoWiredStateStoredAggregateTest {
+class AutoWiredStateStoredAggregateTest {
```

Comment:
I assume there's no equivalent of this in the `hibernate-6-integrationtests` module, as that module doesn't include Spring, right?

Reply:
Ah, I wasn't aware anymore this was a `SpringAxonAutoConfigurer`-specific test. Given the fact the `SpringAxonAutoConfigurer` is deprecated, I think it's fine to not move this test class over.